packages/timeline-state-resolver-types/src/index.ts

```diff
@@ -69,6 +71,7 @@ export enum DeviceType {
 	VMIX = 20,
 	OBS = 21,
 	TELEMETRICS = 22,
+	TRICASTER = 24,
```

Comment:
Any specific reason for using 24, and not just continue counting?

Reply:
There's already device 23 upstream https://github.com/nrkno/sofie-timeline-state-resolver/blob/release49/packages/timeline-state-resolver-types/src/index.ts#L75-L76

Reply:
This is a good example as to why it's not ideal to use numbers as Enum values